### PR TITLE
Added tally ho to rainbowkit

### PR DESCRIPTION
**Pull request by LeifuChen**
## Description
Add `Tally Ho` to the wallet list.
> Note that `Tally Ho` needs to be set as a default wallet in the setting.
image

## Related issue
#1402

## How Has This Been Tested?
1. Open the page and click on the `connect wallet`
2. Select `Tally Ho` and confirm on the pop-up

## Screenshots (if appropriate):
image

### CODE REVIEW

File: components/Rainbowkit/Tally/Tally.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+};
+
+const Tally = ({ chains, shimDisconnect }: TallyOptions): Wallet => ({
+	id: 'tally',
+	iconBackground: '#FFF',
+	name: 'Tally Ho',
+	iconUrl: async () => TallyIcon,
+	downloadUrls: {
+		android: 'https://play.google.com/store/apps/details?id=io.gnosis.Tally',
+		ios: 'https://apps.apple.com/us/app/gnosis-Tally/idid1515759131',
```
Comment (platschi) [inline components/Rainbowkit/Tally/Tally.ts:18]: These links don't work, should probably set them to empty strings as Tally does not have Android/iOS apps?

**Review by platschi — CHANGES_REQUESTED** [review]

**Review by platschi — COMMENTED** [review]
Works good with Tally Ho in Brave Browser. Two points:

- Switching to Optimistic Goerli / Goerli does not work (looks like Tally does not support them?) - no blocker though for mainnet usage, just an observation
- The round Tally logo makes it look not in line with the other logos in the rainbow onboarding modal. Can we update it, maybe change the white background of the logo to match the Tally bg color?

File: components/Rainbowkit/Tally/Tally.ts (hunk 1 above)
Comment (LeifuChen) [reply components/Rainbowkit/Tally/Tally.ts:18]: Yes. Good catch. I will update these links.

**Comment by Tburm** [comment]
Pushing a commit to fix the icon. We needed to specify a background color @LeifuChen @platschi

File: components/Rainbowkit/Tally/Tally.ts (hunk 1 above)
Comment (Tburm) [reply components/Rainbowkit/Tally/Tally.ts]: I pushed a commit to remove these links

**Review by Tburm — DISMISSED** [review]

**Comment by LeifuChen** [comment]
> Pushing a commit to fix the icon. We needed to specify a background color @LeifuChen @platschi

@Tburm Thanks for the fix.

File: components/Rainbowkit/Tally/Tally.ts (hunk 1 above)
Comment (LeifuChen) [reply components/Rainbowkit/Tally/Tally.ts]: @Tburm I added the link to `Tally Ho`'s chrome browser extension.  
image

**Review by Tburm — APPROVED** [review]

**Comment by platschi** [comment]
Perfect, works like a charm now!

**Review by platschi — APPROVED** [review]

**Pull request merged**